### PR TITLE
Expe mathias

### PR DESCRIPTION
**Pull request by MathiasPaulin**
CMakefiles are now of better quality (but yet improvable), externals submodules may be used easily. Only tested on MacOsX.

### CODE REVIEW

File: Applications/MainApplication/CMakeLists.txt [hunk 1]
```diff
@@ -15,26 +16,26 @@ set( Qt5_LIBRARIES
...
 
+# TODO, do we really need this ?
+#if(APPLE)
+#    FIND_LIBRARY(CARBON_LIBRARY Carbon)
+#    set( PLATFORM_LIBS ${CARBON_LIBRARY} )
+#elseif( UNIX )
+#    set( PLATFORM_LIBS X11 xcb )
+#else()
+#    set( PLATFORM_LIBS "" )
+#endif()
```
Comment (nmellado) [inline Applications/MainApplication/CMakeLists.txt]: TODO: remove instead of comment

Comment (MathiasPaulin) [inline Applications/MainApplication/CMakeLists.txt]: DONE

**Comment by MathiasPaulin** [comment]
Hello,
do not make the review yet for this pull request. When trying to integrate glbinding, I found some more problems that need to change this pull request.

**Comment by MathiasPaulin** [comment]
This pull request is obsolete ... I close it.

**Pull request closed without merging**